HTTPSConnectionPool(host='api-inference.huggingface.co', port=443): Max retries exceeded with url: /pipeline/feature-extraction/sentence-transformers/LaBSE (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f5c521d93a0>: Failed to resolve 'api-inference.huggingface.co' ([Errno -2] Name or service not known)"))
ERROR: cannot make dir /home/h4438/Desktop/graduate/WebQA/core/instanceERROR: cannot make dir /home/h4438/Desktop/graduate/WebQA/core/instanceERROR: cannot make dir /home/h4438/Desktop/graduate/WebQA/core/instanceERROR: cannot make dir /home/h4438/Desktop/graduate/WebQA/core/instanceERROR: cannot make dir /home/h4438/Desktop/graduate/WebQA/core/instanceERROR: cannot make dir /home/h4438/Desktop/graduate/WebQA/core/instanceERROR: cannot make dir /home/h4438/Desktop/graduate/WebQA/core/instanceERROR: cannot make dir /home/h4438/Desktop/graduate/WebQA/core/instanceERROR: cannot make dir /home/h4438/Desktop/graduate/WebQA/core/instanceERROR: cannot make dir /home/h4438/Desktop/graduate/WebQA/core/instanceERROR: cannot make dir /home/h4438/Desktop/graduate/WebQA/core/instance[ERROR] 0ERROR: cannot make dir /home/h4438/Desktop/graduate/WebQA/core/instance[ERROR] __init__() got an unexpected keyword argument 'status'ERROR: cannot make dir /home/h4438/Desktop/graduate/WebQA/core/instance